### PR TITLE
Adding other useless folders in .gitignore

### PR DESCRIPTION
**Pull request by FanJups**
I've just added useless files from NetBeans and VS Code.

```
# NetBeans
/nbproject/private/
/nbbuild/
/nbdist/
/.nb-gradle/

#VS Code
.vscode/
```

### CODE REVIEW

**Comment by NiteshKant** [comment]
@servicetalk-bot test this please

File: .gitignore [hunk 1]
```diff
@@ -47,3 +56,9 @@ jenkins/
 # Temp files for scripts/update-copyright.sh
 /update-copyright-modified-files
 /update-copyright-temp
+
```
Comment (Scottmitch) [inline .gitignore:59]: can you remove these extra newlines?

**Comment by Scottmitch** [comment]
@servicetalk-bot test this please

**Comment by FanJups** [comment]
You're welcome @idelpivnitskiy !

**Pull request merged**